app/models/job_proxy_dispatcher.rb

```diff
@@ -246,11 +242,11 @@ def concurrent_job_max_by_proxy(proxy)
   def busy_proxies
     @busy_proxies_hash ||= begin
       Job.where(:dispatch_status => "active")
-      .where("state != ?", "finished")
-      .select([:agent_id, :agent_class])
+         .where("state != ?", "finished")
+         .select([:agent_id])
       .each_with_object({}) do |j, busy_hsh|
-        busy_hsh["#{j.agent_class}_#{j.agent_id}"] ||= 0
-        busy_hsh["#{j.agent_class}_#{j.agent_id}"] += 1
+        busy_hsh["MiqServer_#{j.agent_id}"] ||= 0
+        busy_hsh["MiqServer__#{j.agent_id}"] += 1
```

Comment:
Typo...double underscore should be a single underscore.  This also implies this code does not have testing.